app/dbmodels.py

```diff
@@ -42,9 +39,58 @@ class Meta(BaseMeta):
     settings: pydantic.Json = ormar.JSON()
 
 
+class Conversations(ormar.Model):
```

Comment:
```suggestion
class Conversation(ormar.Model):
```
Названия классов лучше указывать в единственном числе, они описывают каждый объект хранящийся в таблице. 
А уже названия таблиц да - во множественном.

Не то что бы критично, просто было бы неплохо соблюдать один принцип именования)